src/cli.rs

```diff
@@ -74,6 +77,12 @@ impl Options {
                 .multiple(true)
                 .conflicts_with("q")
                 .help("Increases the level of verbosity (the max level is -vvv)"))
+            .arg(Arg::with_name("command")
+                .short("e")
+                .multiple(true)
+                .takes_value(true)
+                .allow_hyphen_values(true)
+                .help("Command and args to execute (must be last argument)"))
```

Comment:
Maybe we can take one string value instead of multiple values with allowed hyphen? Like -e 'tmux new-session -s ssh'